models.md

```diff
@@ -0,0 +1,35 @@
+# Models definition
++ The store needs to be able to map type names to TS classes.
+
+### Proposition 1
+
+```typescript
+@JsonApiModel({
+  type: 'posts',
+  endpoint: '/posts',
+  baseUrl: 'http://api.example.com'
```

Comment:
Yeah, the idea is that you could do some global config on the store itself, but still be able to override stuff at model level (for third party services for instance).